### ISSUE TITLE
Folder mapping issue with dot

### ISSUE THREAD

**Issue by sag3ll0** [mkloubert/vs-deploy#122]
I have package
`
"packages": [           
            {
                "name": "Api",
                "description": "Deploy Api",
                "files": [
                    "../Q.API/Areas/**",
                    "../Q.API/bin/**",
                    "../Q.API/Content/**",
                    "../Q.API/fonts/**",
                    "../Q.API/Scripts/**",
                    "../Q.API/Views/**",
                    "../Q.API/ApplicationInsights.config",
                    "../Q.API/Global.asax",
                    "../Q.API/NLog.config",
                    "../Q.API/Web.config"
                ]
            }
        ],
` 
and target
`
"targets": [
            {
                "type": "local",
                "name": "Local Folder",
                "description": "My Local Folder",
                "dir": "./out",
                "mappings": [
                    {
                        "source": "/wwwroot",
                        "target": "/"
                    },
                    {
                        "source": "/Q.API",
                        "target": "/api"
                    }
                ]
            }]
`
when i execute this deploy  i got this
![image](https://user-images.githubusercontent.com/18505406/32680814-929151bc-c67d-11e7-8521-c8410e951f07.png)
why folder name ".API" not simple "api". Can you check work with dot?

**Comment by mkloubert** [comment]
@sag3ll0 

Sorry for the late answer!

I tried to reproduce the issue, but I found another problem: When I look at your config, it seems to be, that you try to deploy files outside your workspace.
This is not possible, because you will get a "relative path could not be resolved" error for each file outside your workspace.

**Comment by sag3ll0** [comment]
No, all files is in my workspace, but destination if outside. ".." - its folder with visual studio solution and "Q.API" its one of projects.
